tests/go/GoVetBearTest.py

```diff
@@ -24,3 +25,25 @@ def setUp(self):
     def test_run(self):
         self.check_validity(self.uut, [], self.good_file)
         self.check_invalidity(self.uut, [], self.bad_file)
+
+    def test_args(self):
+        self.section.append(Setting('disallow_assembly_go_mismatches', True))
+        self.section.append(Setting('check_useless_assignments', True))
+        self.section.append(Setting('check_sync_package_mistakes', True))
+        self.section.append(Setting('check_boolean_operator_mistakes', True))
+        self.section.append(Setting('disallow_bad_build_tags', True))
+        self.section.append(Setting('disallow_cgo_pointer_violations', True))
+        self.section.append(Setting('disallow_unkeyed_composites', True))
+        self.section.append(Setting('disallow_copying_locks', True))
+        self.section.append(Setting('check_method_signatures', True))
+        self.section.append(Setting('disallow_nil_function_comparisons', True))
+        self.section.append(Setting('check_printf_calls', True))
+        self.section.append(Setting('check_range_loop_variables', True))
+        self.section.append(Setting('check_shadow_variables', True))
+        self.section.append(Setting('check_shifts', True))
+        self.section.append(Setting('check_struct_tag_format', True))
+        self.section.append(Setting('disallow_unreachable_code', True))
+        self.section.append(Setting('disallow_unsafe_pointers', True))
+        self.section.append(Setting('disallow_unused_results', True))
```

Comment:
I think you have enabled all checks. Could you add more tests by disabling some of the checks.

Reply:
Actually, all of these checks are already enabled in `vet`. This PR doesn't add the functionality to disable any check explicitly. The way to disable them is by setting the other flags to `true`. That is if you enable a particular flag explicitly, that check is performed and all others are neglected. Note: They do not hinder with each other (multiple flags can be set to `true`).

Reply:
Another test can be added by enabling a subset of them, but IMO will not add any value. Your say?

Reply:
I'm aware of it.
> Actually, all of these checks are already enabled in vet.

This is what IMO should be tested too. We add tests to check all the cases which we are going to provide via the Bear.
> The way to disable them is by setting the other flags to `true`

Yes, It will not add any further value but it will check other possibilities of user's settings. I hope you are understanding my concern.
> Another test can be added by enabling a subset of them, but IMO will not add any further value.